### PR TITLE
Lazperf integration

### PR DESCRIPTION
**Pull request by ibell13**
Reads LAS & LAZ with code from hobuinc/laz-perf. Removed hexer LAS processing.

### CODE REVIEW

File: hexer/Processor.cpp [hunk 1]
```diff
@@ -82,6 +84,31 @@ void process(HexGrid *grid, PointReader reader)
     grid->findParentPaths();
 }
 
+void processLaz(HexGrid *grid, std::ifstream& file)
+{
+    char buf[30];
```
Comment (abellgithub) [inline hexer/Processor.cpp:89]: Some LAS/LAZ files will have point records greater than 30 bytes. You'll have to get the size from the header and make a buffer that's the right size. In C++ you would normally use a std::vector as the buffer when you don't know the size until runtime. There are lots of examples of this around, but let me know if you have trouble.

File: hexer/Processor.cpp [hunk 2]
```diff
@@ -86,13 +86,14 @@ void process(HexGrid *grid, PointReader reader)
...
 void processLaz(HexGrid *grid, std::ifstream& file)
 {
-    char buf[30];
-
     lazperf::reader::generic_file l(file);
 
-    const lazperf::header14 h = l.header();
     size_t count = l.pointCount();
 
+    const lazperf::header14 h = l.header();
```
Comment (abellgithub) [inline hexer/Processor.cpp:93]: This isn't guaranteed to work in C++. It may work with your compiler, but it's not valid in general. Use a std::vector.

Comment (ibell13) [reply hexer/Processor.cpp:93]: OK. I'll have to change lazperf's readPoint() then (only accepts char arrays)

**Comment by abellgithub** [comment]
Nice.

File: CMakeLists.txt [hunk 3]
```diff
@@ -87,7 +71,7 @@ if (GDAL_FOUND)
 endif()
 
 if (NOT WIN32)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```
Comment (hobu) [inline CMakeLists.txt:90]: This is old. You should use the CMake construction for this

```
set(CMAKE_CXX_STANDARD 17)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

File: apps/curse.cpp [hunk 4]
```diff
@@ -23,8 +23,9 @@
 #include <hexer/HexIter.hpp>
 #include <hexer/Processor.hpp>
 #include <hexer/Utils.hpp>
-#include "las.hpp"
 #include "ProgramArgs.hpp"
+#include "../lazperf/readers.hpp"
```
Comment (hobu) [inline apps/curse.cpp:27]: You should set the include paths in the CMake configuration instead of depending on these files being in relatives paths.

**Pull request merged**